sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -303,7 +303,7 @@ private[hive] class HiveClientImpl(
     // with the side-effect of Hive.get(conf) to avoid using out-of-date HiveConf.
     // See discussion in https://github.com/apache/spark/pull/16826/files#r104606859
     // for more details.
-    Hive.get(conf)
+    shim.getHive(conf)
```

Comment:
Here, line 303 claims that we need `the side-effect of Hive.get(conf)`.
Could you confirm that `shim.getHive(conf)` doesn't break the side-effect assumption?

Reply:
yeah it shouldn't - both function are doing the same in this case by updating the `Hive` object's config with provided `conf`.